api/src/main/java/com/cloud/event/EventTypes.java

```diff
@@ -87,6 +87,7 @@
 
 public class EventTypes {
 
+    public static final String EVENT_VMSCHEDULE_CREATE = "VM.SCHEDULE";
```

Comment:
is there no event for delete or update?

Reply:
Event added for delete,enable and disable in latest commit.